packages/manager/apps/pci-block-storage/src/components/Encryption.tsx

```diff
@@ -47,9 +47,14 @@ const Radio = ({
 interface Props {
   encryptionType: EncryptionType | null;
   onChange: (value: EncryptionType | null) => void;
+  texts: {
```

Comment:
I don't understand why creating a sub-object is useful here, also the name is really generic

Having each property directly at root level seems to be cleaner